jwql/website/apps/jwql/monitor_views.py

```diff
@@ -157,10 +157,9 @@ def claw_monitor(request):
     template = "claw_monitor.html"
 
     # Get all recent claw stack images from the last 10 days
-    query = NIRCamClawStats.objects  # .filter(expstart_mjd__gte=(Time.now().mjd - 10))
-    query = query.order_by('-expstart_mjd').all().values('expstart_mjd', 'skyflat_filename')
-    df = pd.DataFrame.from_records(query)
-    recent_files = list(pd.unique(df['skyflat_filename'][df['expstart_mjd'] > Time.now().mjd - 10]))
+    query = NIRCamClawStats.objects.filter(expstart_mjd__gte=(Time.now().mjd - 10))
```

Comment:
I see the changes now @york-stsci thank you!

Reply:
Yeah. There's still something about how the data frame is built from the query that means that, even though the view only uses the filename field (given that it's already filtering by expstart), it seems like I have to retrieve both fields in order to then filter by uniqueness.